content/committees/steering/brand_use_by_projects.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Proper use of the Jakarta® Brand by Eclipse Projects"
+date: 2020-03-03T13:10:00-04:00
+---
+
+A critical aspect of managing a brand is avoiding confusion as to what the brand represents, what expectations should we expect to be met when something is labeled with that brand. It is important to clearly delineate what we mean when we label something “Jakarta”, that is, when we use “Jakarta” as a modifier/adjective to an object/noun (e.g., “Jakarta® Faces Specification”),  and managing how it is used in the project space is part of this. Careful use of the name will help the community better understand what is directly represented by the Jakarta brand and what is not.
+
+Specifications developed under the purview of the Jakarta EE Specification Committee must use the Jakarta brand in adjective form in their specification names (i.e., “Jakarta® [EE] X”). For example, the “Jakarta® EE Full Platform” or “Jakarta® Enterprise Beans”. All other uses of the Jakarta brand must use “Jakarta” as the object (i.e., “X for Jakarta® [EE]”). Including “EE” in either form is considered optional (and left to the discretion of the project team).
```

Comment:
"All other uses" is too broad since clearly any use that's a reference to another Jakarta spec by name should
use the adjective form.